### PR TITLE
Simple login button ready and fully functional

### PR DESCRIPTION
**Pull request by Joshibbotson**
Screenshot 2023-04-26 at 19 52 57
login via google fully functional

input is not functional yet and is a placeholder in case we set up login via email and password.

### CODE REVIEW

File: src/pages/login/Login.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        if (loading) {
+            return;
+        }
+        if (user) {
+            console.log("user active");
+        }
+    }, [user, loading]);
+
+    return (
+        <>
```
Comment (Gw3i) [inline src/pages/login/Login.tsx:29]: No need for the fragments, when wrapping with a container

File: src/pages/login/Login.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+            <StyledLoginContainer>
+                <StyledLoginBox>
+                    <StyledTextContainer>
+                        <StyledTitleRowContainer>
+                            <h1>Sign in</h1>
+                            <a href="#">I don&apos;t have an account</a>
```
Comment (Gw3i) [inline src/pages/login/Login.tsx:35]: Why are there Predefined Character Entities?

File: src/pages/login/Login.tsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+                        </StyledTitleRowContainer>
+                        <input placeholder="Email" type="text" />
```
Comment (Gw3i) [inline src/pages/login/Login.tsx:37]: type="email"

**Review by Gw3i — CHANGES_REQUESTED** [review]

File: src/pages/login/Login.tsx (hunk 2 above)
Comment (Joshibbotson) [reply src/pages/login/Login.tsx:35]: Screenshot 2023-04-27 at 17 51 32


To fix this issue

**Review by Gw3i — APPROVED** [review]

**Pull request merged**